scripts/gen-js-protos.sh

```diff
@@ -0,0 +1,36 @@
+LIB_VERSION=$1
```

Comment:
Simple script for generating the node module containing the js grpc bindings

Reply:
Cool 🧙